src/sass/components/_buttons.scss

```diff
@@ -81,6 +80,7 @@ Styleguide 2.0
 
   &:link{
     color: $link;
+    text-decoration: underline;
```

Comment:
i added the explicit one because if you apply `.btn--link` to a `<button>`, you wouldn't otherwise get the outline